README.md

```diff
@@ -2,216 +2,267 @@
+You might have `SkyUI_5_1-3863-5-1.7z` and `SkyUI_5_1-3863-5-1.7z.meta`.
+
+These `.meta` files are created by MO2 and typically get created when you click the _MOD MANAGER DOWNLOAD_ button on Nexus Mods. If you open the `.meta` file you will find some metadata for the accompanied archive, the most important entries being `fileID`, `modID` and `gameName`.
+
+Wabbajack will use the MO2 `.meta` files to find out where the archive came from using the [Nexus Mods API](https://app.swaggerhub.com/apis-docs/NexusMods/nexus-mods_public_api_params_in_form_data/1.0#/). This means that you **should** use the _MOD MANAGER DOWNLOAD_ button on Nexus Mods or you have to create those `.meta` files yourself (you can also use the _Query Info_ context menu option in the MO2 Downloads tab. This will populate the metadata of the selected archive).
+
+For more information about `.meta` files, support sites and how to actually create them, see the [Meta File](#meta-files) section.
+
+#### Script Extender and Cleaned Masters
+
+The first step for modding a Bethesda game: Download the game, install the Script Extender and clean base game master files (`.esms`). Wabbajack can, of course, automate these steps.
+
+_(You should use MO2s virtual file system for SKSE Scripts, it is not recommended to install the SKSE Scripts into the game folder directly)_.
+
+Wabbajack can match files from your game folder with your downloads folder (more in-depth info on that topic: [Game Folder Files and MO2](#game-folder-files-and-mo2)), this means you download the Script Extender to your MO2 downloads folder and create a `.meta` file for it. The meta file should look something like this:
+
+```ini
+[General]
+directURL=http://skse.silverlock.org/beta/skse64_2_00_18.7z
+```
+
+In the example above we used [SKSE SE 2.0.18](http://skse.silverlock.org/) and you'd have to replace the URL with the direct download link of the Script Extender you are using.
+
+Next up are Cleaned Master Files. Use xEdit through MO2 and clean the base game master files normally, copy the cleaned master files to a new mod in MO2 (called `Cleaned Master Files` or something) and replace the cleaned master files in your game folder with the backup created by xEdit to revert them to their default unmodified state.
+
+#### Game Folder Files and MO2
+
+You now know about `.meta` files and understand that every file has to come from somewhere, this also means that you also need to create `.meta` files for MO2 and files in your game folder. We recommend using the [MO2 GitHub Releases](https://github.com/ModOrganizer2/modorganizer/releases/) instead of the ones you find on the Nexus.
+
+Wabbajack will not only index your entire MO2 folder but also your entire Game folder and will try to also match files from your downloads folder with your game folder. These matched files will end up in a `Game Folder files` folder in the MO2 subdirectory after a user installs your list. Let's look at an example for better understanding:
```

Comment:
```suggestion
Wabbajack will not only index your entire MO2 folder but also your entire game folder and will try to also match files from your game folder to your MO2 folder, if it can't find a source in your downloads and if they are changed save those changes. For mods that need to be installed in the game folder by the user you will have to create a 'Game Folder Files' folder in your MO2 main directory, which is where the `ModOrganizer.exe` is located. Let's look at an example for better understanding:
```